docs/Universe/pabbly.md

```diff
@@ -0,0 +1,181 @@
+title: Flotiq Pabbly integration | Flotiq docs
+description: Flotiq allows you to integrate your content with any system easily.
+
+# Flotiq and Pabbly integration
+
+## Introduction
+
+For the purposes of this tutorial, we will create a pipeline that sends an email with the invitation for new event, when
+new event is created. We are going store the email list and the list of events in Flotiq cms.
+Unfortunately there is no dedicated Pabbly Connect trigger for Flotiq yet, so in this tutorial we will use Pabbly
+webhook trigger with [Flotiq custom webhooks](https://flotiq.com/docs/panel/webhooks/){target="_blank"}.
+
+## Preparation
+
+To start working on the sample integration of Flotiq with Pabbly Connect we need to start by creating two CTDs (content
+type definitions) and some sample COs (content objects)
+
+Let's start by preparing the CTD `Event`. This CTD is one of the blueprints we have predefined, so all you have to do is
+select it from the list and the CTD will be created automatically
+
+![](images/pabbly/pabbly-add-ctd-in-flotiq-1.png){: .center .width75 .border}
+
+There is nothing left to do but save our CTD for the event and move on to the next step. Don’t forget to Save your
+changes.
+
+![](images/pabbly/pabbly-add-ctd-in-flotiq-2.png){: .center .width75 .border}
+
+Now we will create a CTD for the mailing list from which we will take the email addresses to which we will send our
+invitations.
+
+![](images/pabbly/pabbly-add-ctd-in-flotiq-3.png){: .center .width75 .border}
+
+We will store email, first name, last name and date of birth in our mailing list. After creating all the fields, click "
+Save".
```

Comment:
```suggestion
We will store the email, first name, last name, and date of birth on our mailing list. After creating all the fields, click "
Save".
```
Missing `the`, and in Polish we would use `in`, but in English `on` will be more fitting